runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -2885,6 +2885,7 @@ ClientSessionData::processUnloadedClasses(JITaaS::J9ServerStream *stream, const
    {
    if (TR::Options::getVerboseOption(TR_VerboseJITaaS))
       TR_VerboseLog::writeLineLocked(TR_Vlog_JITaaS, "Server will process a list of %u unloaded classes for clientUID %llu", (unsigned)classes.size(), (unsigned long long)_clientUID);
+   std::unordered_map<TR_OpaqueClassBlock*, ClassUnloadedData> unloadedClassMap;
```

Comment:
Does this have to be a hashtable? Wouldn't a vector be a better choice?

Reply:
Also, when using a vector, consider the usage of `reserve()` to avoid repeated allocations. You know the final size which the same as `classes.size()`